site/assets/scss/_skippy.scss

```diff
@@ -1,17 +1,13 @@
 .skippy {
-  display: block;
-  padding: 1em;
+  position: fixed;
+  top: map-get($spacers, 2);
```

Comment:
Actually, @ffoodd can you remove `map-get`s and use the values directly? We don't do this anywhere else.